src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -363,6 +406,11 @@ inline void copy32(UInt8 * dst, const UInt8 * src)
     memcpy(dst + 16, src + 16, 16);
 #endif
 }
+/*  Copy exactly 32 bytes from src to dst.
+    – Uses one 256-bit move when AVX2 is available,
+      otherwise falls back to two 128-bit SSE moves,
+      or portable memcpy.                                            */
```

Comment:
This comment could have been written by AI or inside a text processor like MS Word.